web/client-api/src/main/java/io/deephaven/web/client/api/subscription/SubscriptionTableData.java

```diff
@@ -43,7 +44,7 @@ private interface ArrayCopy {
     private RangeSet index;
 
     // mappings from the index to the position of a row in the data array
-    private TreeMap<Long, Long> redirectedIndexes;
+    private TreeMap<Long, Integer> redirectedIndexes;
```

Comment:
revert this whole file, this is a hack to test a perf issue with un-downsampled charts